src/Simulation/TargetDefinitions/Tests/Type1/Tests.Type1.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.Quantum.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <QscVerbosity>detailed</QscVerbosity>
+    <QSharpDocsGeneration>false</QSharpDocsGeneration>
+    <!-- we will provide our own -->
```

Comment:
Consider restructuring such that it is clear which line this comment is applicable to (the line above or to the line below or other).